CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 ## 1.19.0-rc1 Enterprise
 ### February 20, 2025
 
+**Enterprise LTS:** Vault Enterprise 1.19 is a [Long-Term Support (LTS)](https://developer.hashicorp.com/vault/docs/enterprise/lts) release.
```

Comment:
Can you double check that this verbiage matches the 1.16 verbiage from when it was also released to CE?

Reply:
The first time we tagged 1.16.x as an LTS was in April 24,2024, for the 1.16.2 minor release so I assumed it was ok to tag at the RC since it was probably due to timing